src/Api/getVaccineInfo.ts

```diff
@@ -1,8 +1,8 @@
 import Axios from "axios";
 
-import { Payload } from "../Interfaces/payload";
+import { requestPayload } from "../Interfaces/payload";
 
-export async function getVaccineResults(values: Payload) {
+export async function getVaccineResults(values: requestPayload) {
```

Comment:
It also might be a good idea to make types for response in here (for both success and failure). That way you can ensure that the rest of the setup is working properly (as long as the response fits types). For this project it's not too significant but for bigger ones it helps hugely to ensure that data is passed properly.